Source/WebCore/platform/graphics/transforms/TransformationMatrix.cpp

```diff
@@ -419,103 +419,131 @@ static bool decompose4(const TransformationMatrix::Matrix4& mat, TransformationM
     result.translateZ = localMatrix[3][2];
     localMatrix[3][2] = 0;
 
-    // Vector4 type and functions need to be added to the common set.
-    Vector3 row[3], pdum3;
+    // Note: Deviating from the spec in terms of variable naming. The matrix is
```

Comment:
Should we file a PR on the spec?

Reply:
https://github.com/w3c/csswg-drafts/issues/3712